### DISCUSSION TITLE
Zróżnicowanie czasu wysyłki dla różnych rynków Allegro

### DISCUSSION THREAD

**Discussion by DanielCyran** [allegro/allegro-api#7751] [Suggestion - verification]
Witam,
W naszej firmie aktualnie procesy są zorganizowane w taki sposób, że potrzebujemy mieć możliwość wydłużenia czasu wysyłki paczek do Czech o jeden lub dwa dni.

Nie wszystkie magazyny obsługują wysyłki międzynarodowe i nie pozwalają na wysyłki międzynarodowe. Stąd konieczność wewnętrznych przesunięć międzymaganyzownych co powoduje, że ciężko w wielu przypadkach zaoferować czas wysyłki w takim samym czasie jak wysyłki do klientów z Polski. I to dotyczy każdych krajów, nie tylko Czech.

Obecnie istnieje możliwość ustawienia jednego czasu dostawy dla dwóch rynków. Możemy więc ustawić czas dostawy na trzy / cztery dni, ale nie chcemy zwiększać czasu wysyłki dla Polski o jeden lub dwa dni.

W związku z tym prosimy o możliwość oddzielenia czasu dostawy dla Polski i dla Czech.

**Comment by MartaNowaczyk** [comment]
Zamykam wątek ze względu na [duplikat](https://github.com/allegro/allegro-api/discussions/7752). Pozostawiłam późniejszy wątek ze względu na komentarze.
